app/components/list-item/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react'
+import ListItem from '.'
+import { IListItemType } from '@/app/app-interfaces'
+import { faker } from '@faker-js/faker'
+import { appLabels } from '@/app/constants'
+
+describe('List-item Component', () => {
+  const todoItems: IListItemType[] = [
+    {
+      id: faker.string.uuid(),
+      name: faker.word.words(),
+    },
+    {
+      id: faker.string.uuid(),
+      name: faker.word.words(),
+    },
+  ]
+  test('render list-item component', () => {
+    render(<ListItem items={todoItems} />)
+    const listItemEl = screen.getByTestId('list-item-wrapper')
```

Comment:
testing by ID must be used as last resolve. You can check the documentation here.

[https://testing-library.com/docs/queries/about/#priority](https://testing-library.com/docs/queries/about/#priority)